components/ProjectsGrid/styles/ProjectsLayout.module.scss

```diff
@@ -0,0 +1,64 @@
+.ProjectsLayout {
+  padding: 150px 0 100px 0;
+  cursor: default;
+
+  .projectDiv {

```

Comment:
Sakyciau is didziosios elementu pavadinimus rasyt - be consistent

Dar pats classname kitoks galetu but? ProjectWrapper, ProjectContainer etc.
Kadangi naudoji CSS Modules 'Project' isvis gali nenaudot. Tai gali vadint tsg Wrapper, Container, Info etc.